### DISCUSSION TITLE
Web SDK能否支持小程序？

### DISCUSSION THREAD

**Discussion by zhisenyang** [Tencent/libpag#55] [Q&A] [answered]
Web SDK能否支持小程序？
准备尝试下，因为Web SDK用了WebAssembly，不知道能否运行起来。

**Comment by zenoslin** [comment, marked as answer]
感谢你的支持，微信小程序版本PAG SDK已经在适配中了。
小程序支持WebAssembly，但是运行环境和标准Web环境存在较大的差异，所以需要一些适配工作。
Web SDK相关的后续更新计划可以关注我们的roadmap：
https://github.com/Tencent/libpag/wiki/PAG-Web-roadmap

**Reply by zhisenyang** [reply]
棒棒！！

**Reply by wangbao2336** [reply]
我想请教一下大佬，小程序适配后pag文件能支持在列表中滚动吗？

**Reply by zenoslin** [reply]
> 我想请教一下大佬，小程序适配后pag文件能支持在列表中滚动吗？

对于列表中滚动的疑问不是很理解，小程序中列表是存在什么限制吗？
小程序适配后pag文件是以canvas的形式存在的，PAG SDK比较聚焦的是canvas上绘制的这一块内容。

**Reply by wangbao2336** [reply]
> > 我想请教一下大佬，小程序适配后pag文件能支持在列表中滚动吗？
> 
> 对于列表中滚动的疑问不是很理解，小程序中列表是存在什么限制吗？ 小程序适配后pag文件是以canvas的形式存在的，PAG SDK比较聚焦的是canvas上绘制的这一块内容。

把pagview放在scroll-view中貌似是不能随着scroll-view的滚动而滚动的，canvas属于原生组件，不能在scroll-view中使用，如果想有这种滚动的效果，有什么好的方法吗？

**Reply by zenoslin** [reply]
@wangbao2336 这块我并不是特别了解，等我们的适配工作完成后，会了解一下看是否会有好的解决方案。

**Reply by wangbao2336** [reply]
> @wangbao2336 这块我并不是特别了解，等我们的适配工作完成后，会了解一下看是否会有好的解决方案。

好的，坐等大佬的完美方案
